addons/wp/stringtable.xml

```diff
@@ -3,777 +3,916 @@
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_Side_DisplayName">
             <English>Side</English>
+            <Czech>Strana</Czech>
             <German>Seite</German>
             <Polish>Strona</Polish>
             <Russian>Сторона</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_Side_Tooltip">
             <English>Which side is calling for artillery</English>
+            <Czech>Která strana volá dělostřelectvo</Czech>
             <German>Die Seite die die Artillerie ruft.</German>
             <Polish>Która strona wzywa ostrzał artyleryjski</Polish>
             <Russian>Какая сторона вызывает артиллерию</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_MainSalvo_DisplayName">
             <English>Main Salvo</English>
+            <Czech>Hlavní salva</Czech>
             <German>Hauptsalve</German>
             <Polish>Główna salwa</Polish>
             <Russian>Главный залп</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_MainSalvo_Tooltip">
             <English>Number of rounds in the main salvo</English>
+            <Czech>Počet střel v hlavní salvě</Czech>
             <German>Anzahl an Schüssen der Hauptsalve.</German>
             <Polish>Liczba pocisków w głównej salwie</Polish>
             <Russian>Количество снарядов в главном залпе</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_Spread_DisplayName">
             <English>Spread</English>
+            <Czech>Rozpětí</Czech>
             <German>Streuung</German>
             <Polish>Rozrzut</Polish>
             <Russian>Разброс</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_Spread_Tooltip">
             <English>Default dispersion of the main salvo</English>
+            <Czech>Výchozí rozptyl hlavní salvy</Czech>
             <German>Standard Streuung der Hauptsalve.</German>
             <Polish>Standardowy rozrzut głównej salwy</Polish>
             <Russian>Стандартный разброс для главного залпа</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_SkipCheckrounds_DisplayName">
             <English>Skip adjusting rounds</English>
+            <Czech>Přeskočit upravování střel</Czech>
             <German>Anpassen der Schüsse überspringen</German>
             <Polish>Pomiń strzały korygujące</Polish>
             <Russian>Пропустить подстройку снарядов</Russian>
         </Key>
         <Key ID="STR_Lambs_WP_Module_TaskArtillery_SkipCheckrounds_Tooltip">
             <English>Check this to disable initial rounds used by the fire controller to adjust rounds on target.\nSkipping this will make the barrage immediately hit on target</English>
+            <Czech>AAAAAAAA</Czech>
```

Comment:
AAAAAAAA? i think there is the translation missing

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

Reply:
Severe issue of AAAAAAAA has been fixed.